sdk/azcore/log.go

```diff
@@ -77,16 +77,13 @@ func (l *Logger) Should(cls LogClassification) bool {
 // Write invokes the underlying Listener with the specified classification and message.
 // If the classification shouldn't be logged or there is no listener then Write does nothing.
 func (l *Logger) Write(cls LogClassification, message string) {
-	if l.lst == nil || !l.Should(cls) {
-		return
-	}
-	l.lst(cls, message)
+	l.Writef(cls, "%s", message)
```

Comment:
The call to `l.lst()` in this case was intentional, to avoid interpreting any `%` characters as format specifiers (URL-encoded query params will contain them).

Reply:
I've revert this.